pkg/scheduler/framework/plugins/noderesources/fit.go

```
@@ -130,10 +130,9 @@ func (f *Fit) PreFilterExtensions() framework.PreFilterExtensions {
 func getPreFilterState(cycleState *framework.CycleState) (*preFilterState, error) {
 	c, err := cycleState.Read(preFilterStateKey)
 	if err != nil {
-		// The preFilterState doesn't exist. We ignore the error for now since
-		// Filter is able to handle that by computing it again.
+		// The preFilterState doesn't exist.
 		klog.V(5).Infof("Error reading %q from cycleState: %v", preFilterStateKey, err)
-		return nil, nil
+		return nil, err
```

Ditto. We can simply return a verbose err and remove the logging.